backend/aws_helpers/dynamo_db_utils/trainspace.py

```diff
@@ -0,0 +1,233 @@
+import json
+
+from dataclasses import dataclass
+from datetime import datetime
+
+from backend.aws_helpers.dynamo_db_utils.base_db import (
+    BaseData,
+    BaseDDBUtil,
+    enumclass,
+    changevar,
+)
+from backend.common.constants import TRAINSPACE_TABLE_NAME, AWS_REGION
+from boto3.dynamodb.conditions import Key
+from backend.common.utils import get_current_timestamp
+from typing import Union
+
+PRIMARY_KEY = "trainspace_id"
+
+REQUIRED_KEYS = [
+    "trainspace_id",
+    "name",
+    "training_file",
+    "uid",
+    "step",
+    "status",
+    "modified",
+    "train_model",
+]
+
+
+@dataclass
+class TrainspaceData(BaseData):
+    """Data class to hold the attribute values of a record of the execution-table DynamoDB table"""
```

Comment:
you mean trainspace table? @farisdurrani

Reply:
Nooo, TrainspaceData. That is the convention we're using in our code